src/components/SingleJudgment.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import {Header} from "semantic-ui-react";
+import {connect} from "react-redux";
+
+class SingleJudgment extends Component {
+  render() {
+    const judgment = this.props.data;
+    const {riders} = this.props;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "riders" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'riders' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)